soc/silabs/common/soc_power_pmgr.c

```diff
@@ -8,6 +8,7 @@
 #include <zephyr/logging/log.h>
 #include <zephyr/pm/pm.h>
 #include <sl_power_manager.h>
+#include <sli_hfxo_manager.h>
```

Comment:
I get a compiler warning on xg24:
```
zephyr/soc/silabs/common/soc_power_pmgr.c:97:9: warning: implicit declaration of function 'sl_hfxo_manager_irq_handler'; did you mean 'sli_hfxo_manager_init_hardware'? [-Wimplicit-function-declaration]
   97 |         sl_hfxo_manager_irq_handler();
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |         sli_hfxo_manager_init_hardware
```

The missing symbol seems to be in `sl_hfxo_manager.h`, so that should be included too.